content/news/posts/2025/PSDevCon2024-Cybersecurity-presentation.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title:  "PSDevCon2024: Cybersecurity – What No One Dares to Say"
+subtitle: "Read a summary of Vincent Guesnard’s cybersecurity presentation at the Developer Conference 2024"
+date:   2025-04-30
+authors: [ prestashop ]
```

Comment:
@coffrep Please add and change the author of the article to Vincent

Reply:
Good to hear about adding Vincent as an author - I've updated the author page on my end and made the change on the post.